pkg/cmd/flags.go

```diff
@@ -127,6 +128,11 @@ func AddConfigFlag(cmd *cobra.Command) {
 	cmd.Flags().StringSlice("config", []string{}, `A comma-separated list of "key=value" pairs, or path to a configuration file containing a newline-separated list of "key=value" pairs.`)
 }
 
+func AddConnectionTypesFlag(cmd *cobra.Command) {
```

Comment:
We can (and should) limit the scope of this function. It's only used in the internal/networking package, so we can declare it there instead (perhaps in command.go). We can also limit the scope of the constants declared in pkg/network/utils.go as well.

Reply:
Thanks! I moved them to `internal/network/command.go`